RELEASENOTES-1.4.docu

```diff
@@ -474,4 +474,12 @@
       invalid generated C when both objects are targets of <tt>each
       .. in ..</tt> expressions
       <bug number="SIMICS-21044"/>.</add-note></build-id>
+  <build-id value="next"><add-note>It's now allowed to cast an expression
+      of a struct type to that same struct type.</add-note></build-id>
```

Comment:
This is DML 1.4 exclusive (or rather, non-`compat_dml12_int` exclusive) as DMLC under 1.2 just allows any cast from or to a struct type to slip by it, thus making GCC pick up the slack of reporting bad casts.